setup.py

```diff
@@ -80,7 +80,7 @@
         'jupyter_core>=4.0,<5.0',
         'jupyter_client>=6.1.7',
         'jupyter_server>=1.7.0',
-        'jupyterlab>=3.0.17',
+        'jupyterlab==3.3.0rc0',
```

Comment:
Did you try 

```suggestion
        'jupyterlab~=3.3.0rc0',
```

If I'm not mistaken this should pick up any release candidate